pacsman/filesystem_dev_client.py

```diff
@@ -0,0 +1,131 @@
+'''
+This filesystem "dummy" client should be used for testing in development
+when a PACS server is not available. All get/fetch operations are O(N) on the number of
+DICOM datasets loaded from the `test_dicom_data` dir.
+
+Example data located in `test_dicom_data` dir:
+ (from www.dicomserver.co.uk).
+
+Patient ID PAT001 "Joe Bloggss" dob 19450703
+    Study ID 1.2.826.0.1.3680043.11.1011
+        Study Date 20180522
+        CT Modality
+            Series ID 1.2.826.0.1.3680043.6.86796.74495.20180522152336.14136.1.23
+                1 Image
+
+Patient ID PAT014 "Erica Richardson" dob 19520314
+    Study ID 1.2.826.0.1.3680043.11.118
+        Study Date 20180518
+        CT Modality
+            Series ID 1.2.826.0.1.3680043.6.51581.36765.20180518132103.25992.1.21
+                5 images
+    Study ID 1.2.826.0.1.3680043.11.118.1
+'''
+import glob
+import logging
+import os
+import shutil
+
+from pydicom import dcmread, Dataset
+from pydicom.valuerep import MultiValue
+
+from dicom_interface import DicomInterface
+
+
+logger = logging.getLogger(__name__)
+
+dicom_datasets = {}
+
+file_dir = os.path.dirname(os.path.abspath(__file__))
+dicom_source_dir = os.path.join(file_dir, 'test_dicom_data')
+
+
+class FilesystemDicomClient(DicomInterface):
```

Comment:
`Filesystem` or `FileSystem`?  It seems like both are legitimate choices but it will get progressively more annoying to change later if we do want to change it.

Reply:
I see a good deal of both online; I'm a fan of compound words so I'll probably just keep `Filesystem`.